integration_tests/features/support/steps.js

```diff
@@ -287,6 +287,21 @@ When(/I mine (\d+) blocks on (.*)/, {timeout: 600*1000}, async function (numBloc
     this.tipHeight += parseInt(numBlocks);
 });
 
+When(/I sync mine (\d+) blocks on (.*)/, {timeout: 1200*1000}, async function (numBlocks, name) {
```

Comment:
What does `sync mine` mean?

Reply:
sync mine = mine some blocks while waiting for all other nodes to keep in sync

The problem here is these two regular expressions look similar to cucumber:
`When(/I mine (\d+) blocks on (.*) /`
       vs.
`When(/I mine (\d+) blocks on (.*) while waiting for all other nodes to keep in sync/`

Maybe this will work:
`When(/I keep all nodes in sync and mine (\d+) blocks on (.*)/`